controller/aks-cluster-config-handler.go

```diff
@@ -762,7 +764,8 @@ func (h *Handler) updateUpstreamClusterState(ctx context.Context, config *aksv1.
 	// check Kubernetes version for update
 	if config.Spec.KubernetesVersion != nil {
 		if aks.String(config.Spec.KubernetesVersion) != aks.String(upstreamSpec.KubernetesVersion) {
-			logrus.Infof("Updating kubernetes version for cluster [%s]", config.Spec.ClusterName)
+			logrus.Infof("Updating kubernetes version to %s for cluster [%s (id: %s)]", aks.String(config.Spec.KubernetesVersion), config.Spec.ClusterName, config.Name)
```

Comment:
```suggestion
			logrus.Infof("Updating kubernetes version for cluster [%s (id: %s)]", config.Spec.ClusterName, config.Name)
```

Reply:
Why are you suggesting a revert? Same question for the above suggestion.